README.md

```diff
@@ -1,19 +1,90 @@
 [![Build Status](https://drone.owncloud.com/api/badges/owncloud/web/status.svg)](https://drone.owncloud.com/owncloud/web)
 [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=owncloud_web&metric=security_rating)](https://sonarcloud.io/dashboard?id=owncloud_web)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=owncloud_web&metric=coverage)](https://sonarcloud.io/dashboard?id=owncloud_web)
 [![web docker image](https://img.shields.io/docker/v/owncloud/web?label=web%20docker%20image&logo=docker&sort=semver)](https://hub.docker.com/r/owncloud/web)
 
+
+  
 # ownCloud Web
-**User interface for ownCloud**
+With ownCloud Web you can manage your ownCloud in your browser.
+
+</div>
+<img width="100%" alt="Screenshot of ownCloud Web" src="https://user-images.githubusercontent.com/26610733/168603602-5ba37b4d-2ee1-413c-9a55-03380c79cd3d.png">
+
+ownCloud Web is a single page, standalone frontend, based on modern web technologies. It brings new features as well as improved user flows and can be deployed independent of the backend server. 
+## Examples
+Here are some examples of what you can do with ownCloud Web:
+- 🗂 **Files**: Upload, download, search and manage files (as you may know it for example from Dropbox, OneDrive, Google Drive etc.).
+- 🌈 **Share** files or whole folders directly with other users on your ownCloud.
+- 🌍 **Links**: Create links and share them with anyone in the world - no login required.
+- 📝 **Write**: Edit your documents with the editor of your choice like ONLYOFFICE, Collabora or Microsoft Word and more.
+- 🤝 **Collaborate** in real-time on documents.
+- 🚀 **Spaces**: You have to manage important projects? Let Spaces, the new special folders, keep order. 
+- 🛟 **Versioning** Saved a wrong version? We have the timemachine you where looking for! Easyily go back in time and restore older versions of your files.
+- 📥 **Drop-folders** Collect files from other people in one folder via a simple link, ex. homework from pupils or photos from your family - no login required.
+- 🔒 **100% privacy**: ownCloud Web is GDPR compliant and can be used only internally or toghether with external people.
+- 🛡 **Secure** ownCloud Web is an open source project which means that you can track every action of the software - no blackbox, no "phone home".
+- 🩼 **Inclusive**: Our goal is to be accessible for kids to seniors and newbies to experts as we are all affected by physical and cognitive limitations depending on our personal situations.
```

Comment:
```suggestion
- ♿ **Inclusive:** Our goal is to be accessible for kids as well as seniors and for newbies as well as experts - since we are all affected by physical and cognitive limitations, depending on our personal situation.
```